builder/utility.go

```diff
@@ -362,6 +364,8 @@ func NewSortableIPAPth(pth string) (SortableIPAPth, error) {
 		} else {
 			ipaPth.dirNameIdx = 0
 		}
+	} else {
+		return SortableIPAPth{}, errors.New("Could not find sortable path string")
```

Comment:
please print the path as well: `errors.New("Could not find sortable path string")` -> `fmt.Errorf("Path is not sortable: %s", pth)`